### DISCUSSION TITLE
Can I get some guidance about how to access the computation of the  inv and logdet in the marginal likelihood?

### DISCUSSION THREAD

**Discussion by Miguel-Hombrados** [cornellius-gp/gpytorch#2465] [Q&A]
I need to modify the computation of the kernel inverse and and logdet in the Gaussian marginal likelihood with exact inference, but in the latest versions it became more obscure to me. Thank you!

**Comment by supersjgk** [comment]
Is this what you want to modify?
https://github.com/cornellius-gp/gpytorch/blob/master/gpytorch/distributions/multivariate_normal.py#L193

https://github.com/cornellius-gp/linear_operator/blob/main/linear_operator/functions/_inv_quad_logdet.py
